### ISSUE TITLE
A11y_MissionControlAndAPIs_ExperimentationGroup_GroupDetails_HighContrast: Keyboard focus is not visible in the list items of the "Select" combo edit boxes in high contrast when navigating using down arrow key

### ISSUE THREAD

**Issue by Bhaskar301101** [facebook/react#22226]
Bug discussion:
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
Environment Details:
Application Name: Mission Control and APIs
#URL: https://mission-control-ci.azurewebsites.net/
Windows Version: Win10 
Chrome version: Version 87.0.4280.88

Prerequisite:
Enable high contrast black mode
Repro Steps:
Navigate to https://mission-control-ci.azurewebsites.net/experiments
Tab till "OneSettings Config" tab.
It should default to "Groups" sub tab
Tab till "+ New OneSettings Config Group" button and hit enter
Tab till "Data Science Owner- Select" combo edit box and expand it
Check if keyboard focus is visible on the list items using down arrow key.
Actual:
Keyboard focus is not visible in the list items of the "Select" combo edit boxes in high contrast when navigating using down arrow key

Similar issue is observed throughout the application where "combo edit box" is present.
Expected:
Keyboard focus should be visible in the list items of the "Select" combo edit boxes in high contrast when navigating using down arrow key
User Impact:
Low vision users will not be able to identify where the keyboard focus is inside the list box.
Recommendation:
Refer repository of bug fixes code snippets 
MAS Reference:
MAS 1.4.3 - Contrast (Minimum)
<img width="576" alt="BUG2" src="https://user-images.githubusercontent.com/89897257/131652886-54efb362-4d6c-49e7-b6d3-6db968894210.png">

**Comment by eps1lon** [comment]
This repository is not affiliated with https://mission-control-ci.azurewebsites.net/%E2%80%8B
